lib/TH/THFilePrivate.h

```diff
@@ -1,3 +1,10 @@
+#include "THGeneral.h"
+
+#ifdef TH_GENERIC_USE_HALF
```

Comment:
Fixed, thanks for pointing out! Not just simpler, but most importantly, correct - as THGeneral.h always defines it. In this case it did not make difference, but it's important to have it in uniform fashion.